js/fitbit-service.js

```diff
@@ -0,0 +1,185 @@
+(function() {
+    'use strict';
+
+    var express = require('express');
+    var app     = express();
+    var fs      = require( 'fs' );
+    var Fitbit  = require( 'fitbit-oauth2' );
+
+    function FitbitService($http) {
+        var service = {};
+        var summary = null;
+        var today = null;
+        
+        // Simple token persist functions.
+        //
+        var tfile = 'fb-token.json';
```

Comment:
How often will you have to refresh this token? If it's more than once a month it's going to be a pain. I've got some ideas about having a remote configuration agent in #48 - maybe you could get the token on another device and pass it to the mirror (which would be way easier than manually updating this all the time)